lib/kubeclient.rb

```diff
@@ -1,21 +1,20 @@
 # frozen_string_literal: true
 
 require 'json'
-require 'rest-client'
 
-require 'kubeclient/aws_eks_credentials'
-require 'kubeclient/common'
-require 'kubeclient/config'
-require 'kubeclient/entity_list'
-require 'kubeclient/exec_credentials'
-require 'kubeclient/gcp_auth_provider'
-require 'kubeclient/http_error'
-require 'kubeclient/missing_kind_compatibility'
-require 'kubeclient/oidc_auth_provider'
-require 'kubeclient/resource'
-require 'kubeclient/resource_not_found_error'
-require 'kubeclient/version'
-require 'kubeclient/watch_stream'
+require_relative 'kubeclient/aws_eks_credentials'
```

Comment:
this should not be necessary and can mess with require hacks, so prefer to avoid

Reply:
Not sure what you refer to with `this`? Is it all the `require` => `require_relative` changes? It's just been merged on master, so needs to reverted if it's not the best idea after all.

@grosser can you point me to where I can read more about this?